components/login/login.scss

```diff
@@ -369,8 +323,7 @@
     .login-body .login-body-content {
         min-width: 414px;
```

Comment:
do we ned this? What about phones with widths less than this. The smallest phone could be 375px;

Reply:
Updated to 375px